tests/fips/openjdk/openjdk_ssh.pm

```diff
@@ -0,0 +1,54 @@
+# SUSE's openjdk fips tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: openjdk expect
+#
+# Summary: FIPS: openjdk
+#          Jira feature: SLE-21206
+#          FIPS 140-3: make OpenJDK be able to use the NSS certified crypto
+#          Case 2: Run Java SSH / Client http://www.jcraft.com/jsch/
+# Maintainer: Yutao Wang <yuwang@suse.com>
```

Comment:
Please provide the ticket number.

Reply:
Please use this format.

`# Tags: poo#39071, poo#105591, poo#105999 ....etc.`